training/tf/parse.py

```diff
@@ -260,8 +261,11 @@ def benchmark(parser):
         end = time.time()
         print("{} pos/sec {} secs".format( 10000. / (end - start), (end - start)))
 
-def main(args):
+def split_chunks(chunks, test_ratio):
+    splitpoint = 1 + int(len(chunks) * (1.0 - test_ratio))
+    return (chunks[:splitpoint], chunks[splitpoint:])
```

Comment:
Maybe do random.shuffle(chunks) first, to help the case where early chunks have weak games and later ones have strong games?

Reply:
Yes, that's a good point.